src/DIRAC/WorkloadManagementSystem/FutureClient/JobMonitoringClient.py

```diff
@@ -0,0 +1,36 @@
+# pylint: disable=import-error
+import os
+
+if os.getenv("DIRAC_ENABLE_DIRACX_JOB_MONITORING", "No").lower() in ("yes", "true"):
```

Comment:
This condition is already applied in src/DIRAC/WorkloadManagementSystem/Client/JobMonitoringClient.py.

In general, I think that we can put all this code in src/DIRAC/WorkloadManagementSystem/Client/JobMonitoringClient.py.

Reply:
This is needed because otherwise sphinx crashes. `diracx` is not a dependency of `DIRAC` so we have no other choice that hide all the `diracx` code behind env variables

Reply:
Can't you add diracx to https://github.com/DIRACGrid/DIRAC/blob/4d534a32168891f9ef6ce258dc41f668f5d2302a/docs/diracdoctools/__init__.py#L8 ?

How does sphinx crash?

Reply:
I'll try thanks !